qa/integration-tests/src/test/java/io/camunda/it/utils/CamundaMultiDBExtension.java

```diff
@@ -154,6 +188,59 @@ private ElasticsearchContainer setupElasticsearch() {
     return elasticsearchContainer;
   }
 
+  private boolean validateSchemaCreation(final String url, final String testPrefix) {
+    final HttpRequest httpRequest =
+        HttpRequest.newBuilder()
+            .GET()
+            .uri(URI.create(String.format("%s/%s*", url, testPrefix)))
+            .build();
+    try {
+      final HttpResponse<String> response = httpClient.send(httpRequest, BodyHandlers.ofString());
+      final int statusCode = response.statusCode();
+      if (statusCode / 100 != 2) {
+        return false;
+      }
+
+      // Get how many indices with given prefix we have
+      final JsonNode jsonNode = objectMapper.readTree(response.body());
+      final int count = jsonNode.size();
+      /*
+       * ES exporter indices are only created, on first exporting, so we expect at least the
+       * harmonized index amount or more (to fight race conditions).
+       */
+      final boolean reachedCount = expectedDescriptors.size() <= count;
+      if (!reachedCount) {
+        LOGGER.debug(
+            "[{}/{}] indices with prefix {} in ES, retry...",
+            count,
+            expectedDescriptors.size(),
+            testPrefix);
+
+        final Iterator<String> stringIterator = jsonNode.fieldNames();
+        final Iterable<String> iterable = () -> stringIterator;
+        final List<String> actualIndices =
+            StreamSupport.stream(iterable.spliterator(), false).toList();
+
+        final List<String> missing =
+            expectedDescriptors.stream()
+                .map(IndexDescriptor::getFullQualifiedName)
+                .filter(name -> !actualIndices.contains(name))
+                .collect(Collectors.toList());
+        LOGGER.debug("Missing indices: {}", missing);
```

Comment:
```suggestion
var expectedIndexNames =  expectedDescriptors.stream()
         .map(IndexDescriptor::getFullQualifiedName)
         .toList();
 assertThat(actualIndices).containsAll(expectedIndexNames)
```

:wrench: it can probably be a bit simpler with assert4j

Reply:
👍🏼 

I changed it. It will now fail with:

```
org.awaitility.core.ConditionTimeoutException: Condition with alias 'Await database and exporter readiness' didn't complete within 1 seconds because assertion condition defined as a Lambda expression in io.camunda.it.utils.CamundaMultiDBExtension [Missing indices] 
Expecting SubList:
  ["processinstanceandflownodeinstancequerytest-camunda-authorization-8.8.0_"]
to contain:
  ["processinstanceandflownodeinstancequerytest-camunda-mapping-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-flownode-instance-8.3.1_",
    "processinstanceandflownodeinstancequerytest-operate-list-view-8.3.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-import-position-8.2.0_",
    "processinstanceandflownodeinstancequerytest-operate-variable-8.3.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-form-8.4.0_",
    "processinstanceandflownodeinstancequerytest-operate-decision-requirements-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-metric-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-incident-8.3.1_",
    "processinstanceandflownodeinstancequerytest-camunda-group-8.8.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-draft-task-variable-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-sequence-flow-8.3.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-task-8.5.0_",
    "processinstanceandflownodeinstancequerytest-camunda-user-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-operation-8.4.1_",
    "processinstanceandflownodeinstancequerytest-operate-message-8.5.0_",
    "processinstanceandflownodeinstancequerytest-operate-event-8.3.0_",
    "processinstanceandflownodeinstancequerytest-camunda-web-session-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-decision-instance-8.3.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-metric-8.3.0_",
    "processinstanceandflownodeinstancequerytest-camunda-tenant-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-decision-8.3.0_",
    "processinstanceandflownodeinstancequerytest-camunda-role-8.8.0_",
    "processinstanceandflownodeinstancequerytest-camunda-authorization-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-batch-operation-1.0.0_",
    "processinstanceandflownodeinstancequerytest-operate-process-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-post-importer-queue-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-import-position-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-job-8.6.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-task-variable-8.3.0_"]
but could not find the following element(s):
  ["processinstanceandflownodeinstancequerytest-camunda-mapping-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-flownode-instance-8.3.1_",
    "processinstanceandflownodeinstancequerytest-operate-list-view-8.3.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-import-position-8.2.0_",
    "processinstanceandflownodeinstancequerytest-operate-variable-8.3.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-form-8.4.0_",
    "processinstanceandflownodeinstancequerytest-operate-decision-requirements-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-metric-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-incident-8.3.1_",
    "processinstanceandflownodeinstancequerytest-camunda-group-8.8.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-draft-task-variable-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-sequence-flow-8.3.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-task-8.5.0_",
    "processinstanceandflownodeinstancequerytest-camunda-user-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-operation-8.4.1_",
    "processinstanceandflownodeinstancequerytest-operate-message-8.5.0_",
    "processinstanceandflownodeinstancequerytest-operate-event-8.3.0_",
    "processinstanceandflownodeinstancequerytest-camunda-web-session-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-decision-instance-8.3.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-metric-8.3.0_",
    "processinstanceandflownodeinstancequerytest-camunda-tenant-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-decision-8.3.0_",
    "processinstanceandflownodeinstancequerytest-camunda-role-8.8.0_",
    "processinstanceandflownodeinstancequerytest-operate-batch-operation-1.0.0_",
    "processinstanceandflownodeinstancequerytest-operate-process-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-post-importer-queue-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-import-position-8.3.0_",
    "processinstanceandflownodeinstancequerytest-operate-job-8.6.0_",
    "processinstanceandflownodeinstancequerytest-tasklist-task-variable-8.3.0_"]
.

	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1006)
	at org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)
	at io.camunda.it.utils.CamundaMultiDBExtension.beforeAll(CamundaMultiDBExtension.java:180)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:101)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:81)
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:103)
	... 6 more
```